styles.css

```diff
@@ -151,6 +154,16 @@ video {
   background-position: center;
 }
 
+#callControls .blur-on {
+  background-color: #fff;
+  color: #000;
+}
+
+#callControls .blur-off {
+  background-color: #f63135;
+  color: #fff;
+}
+
```

Comment:
I know we're not going full design mode here but I think we should avoid red for this button (usually means "danger!"). I like the idea of toggling the background colour to show the change stuck (the white is nice). Here's an example using `cyanLight` and `blueLight` from our company colours for the background and text colour. I added the full word "blur" and made the text a little smaller because "B" might be scary to click if you don't know what B stands for 😅 
<img width="818" alt="CleanShot 2022-01-13 at 10 38 55@2x" src="https://user-images.githubusercontent.com/18265845/149361770-6495740e-963c-4eaa-ba3a-f96bd6db6fed.png">

Reply:
This is way better, thank you! Will update now.